x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/alert_summary_widget.tsx

```diff
@@ -9,7 +9,7 @@ import { EuiLoadingSpinner } from '@elastic/eui';
 import React from 'react';
 import { useLoadAlertSummary } from '../../../../hooks/use_load_alert_summary';
 import { AlertSummaryWidgetProps } from '.';
-import { AlertSummaryWidgetError, AlertsSummaryWidgetUI } from './components';
+import { AlertSummaryWidgetError, AlertsSummaryWidgetCompact } from './components';
```

Comment:
💭 I changed `AlertsSummaryWidgetUI` to `AlertsSummaryWidgetCompact` since, in my next task, I will work on a full-size component (this is a preparation).